### ISSUE TITLE
Bug Report: Supervisor installer pulling bad archtype for Raspi3 armv7

### ISSUE THREAD

**Issue by guice** [home-assistant/supervised-installer#213]
### OS Version

Raspbian GNU/Linux 11 (bullseye)

### System Information

Linux venus-ha 5.15.32-v7+ #1538 SMP Thu Mar 31 19:38:48 BST 2022 armv7l GNU/Linux

### What happened?

I'm attempting a complete fresh install of supervisor using the `.deb` package. The install runs successfully after the `grub` hack. However, supervisor is pulling the 64bit image. Which is, as you'd expect, failing:
```
pi@venus-ha:~/Downloads $ docker ps -a
CONTAINER ID   IMAGE                                                              COMMAND   CREATED          STATUS                     PORTS                                   NAMES
21cd89dc1a9b   ghcr.io/home-assistant/raspberrypi3-64-homeassistant:2022.4.7   "/init"   3 minutes ago    Exited (1) 3 minutes ago                                           homeassistant
```

It's failing on arch type:
```
pi@venus-ha:~/Downloads $ docker run ghcr.io/home-assistant/raspberrypi3-64-homeassistant:2022.4.7
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested
standard_init_linux.go:228: exec user process caused: exec format error
```

I've looked, and according to your hardware specs, you support raspberrypi 3 (non-64).
```
pi@venus-ha:~/Downloads $ cat /proc/cpuinfo 
processor	: 0
model name	: ARMv7 Processor rev 4 (v7l)
BogoMIPS	: 76.80
Features	: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32 
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xd03
CPU revision	: 4

[snip]

Hardware	: BCM2835
Revision	: a22082
Serial		: 00000000fbdf1ae1
Model		: Raspberry Pi 3 Model B Rev 1.2
```

### Machine Type

raspberrypi3

### Installer output

```text
pi@venus-ha:~/Downloads $ sudo dpkg -i homeassistant-supervised.deb 
(Reading database ... 187750 files and directories currently installed.)
Preparing to unpack homeassistant-supervised.deb ...
[warn] 
[warn] If you want more control over your own system, run
[warn] Home Assistant as a VM or run Home Assistant Core
[warn] via a Docker container.
[warn] 
[warn] ModemManager service is enabled. This might cause issue when using serial devices.
Leaving 'diversion of /etc/NetworkManager/NetworkManager.conf to /etc/NetworkManager/NetworkManager.conf.real by homeassistant-supervised'
Leaving 'diversion of /etc/NetworkManager/system-connections/default to /etc/NetworkManager/system-connections/default.real by homeassistant-supervised'
Leaving 'diversion of /etc/docker/daemon.json to /etc/docker/daemon.json.real by homeassistant-supervised'
Leaving 'diversion of /etc/network/interfaces to /etc/network/interfaces.real by homeassistant-supervised'
Unpacking homeassistant-supervised (1.1.1) over (1.1.1) ...
Setting up homeassistant-supervised (1.1.1) ...
[info] Restarting NetworkManager
[info] Restarting docker service
PING version.home-assistant.io (104.26.4.238) 56(84) bytes of data.
64 bytes from 104.26.4.238 (104.26.4.238): icmp_seq=1 ttl=57 time=27.4 ms

--- version.home-assistant.io ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 27.403/27.403/27.403/0.000 ms
Device "eth0
eth0" does not exist.
[info] Install supervisor Docker container
[info] Install supervisor startup scripts
[info] Install AppArmor scripts
[info] Start Home Assistant Supervised
[info] Installing the 'ha' cli
[info] Within a few minutes you will be able to reach Home Assistant at:
[info] http://homeassistant.local:8123 or using the IP address of your
[info] machine: http://:8123
```

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by guice** [comment]
I'm going to close this issue myself. It appears there is a prompt on initial install where you select your arch type. I don't recall it, but maybe it did come up and I selected 64 ? I don't recall. I found this prompt (again?) when I `--purge` the package and re-installed. This time, I _did_ see it and selected the standard raspberrypi 3 and it's now working!

**Comment by alexschwantes** [comment]
hmmm I'm going to add to this as I just went through this exact same problem (ie. installing on rpi3). I already had it installed but had an issue with an upgrade so I ran the latest installer. It didn't ask any prompts to select an arch type and just went about its merry way.. but failed to load. Until I found your post and also confirmed it was installing the x86-64-homeassistant instead of armv7. It was only once I ran `sudo dpkg --purge homeassistant-supervised` and then re-installed that it worked.

So there could be a bug here. 

There is still something off with the installer though as it pulled `raspberrypi3-homeassistant:landingpage` docker image which is only 86mb. I have no idea what that image is or why it kept trying to use it, but it didn't work. But at least it was the right arch and allowed me to manually set the version with `ha core update --version 2022.6.7` which then actually booted as expected..
